python/foglamp/tasks/north/sending_process.py

```diff
@@ -904,7 +904,7 @@ def start(self):
             sys.exit(1)
         try:
             self._storage = StorageClient(self._mgt_address, self._mgt_port)
-            self._readings = ReadingsStorageClient(self._mgt_address, self._mgt_port)
+            self._readings = ReadingsStorageClientAsync(self._mgt_address, self._mgt_port)
```

Comment:
now we need to 
1) await fetch readings
2) use aiohttp client session to POST data and eliminate the need to synchronous request lib